### PR TITLE
Setting the version to match reality. Sym linking py3 to python for parity

### PR DESCRIPTION
**Pull request by coryleeio**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,10 +19,13 @@ MAINTAINER devops@edx.org
 # MySQL-python for performance gains.
 
 # If you add a package here please include a comment above describing what it is used for
-RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python3.5 python3-pip libmysqlclient-dev libssl-dev python3-dev -qy && \
+RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python3.6 python3-pip libmysqlclient-dev libssl-dev python3.6-dev -qy && \
```
Comment (adzuci) [inline Dockerfile]: @coryleeio Do you know if we have tested 3.6 or what is needed to get this PR merged.

**Comment by sarina** [comment]
Closing as @coryleeio has left the Open edX project. The PR can be reopened if work should continue.

**Comment by openedx-webhooks** [comment]
Although this pull request is already closed, I'm still watching it for updates.

There is nothing you have to do. No action is needed from your side.  Thanks.

**Comment by openedx-webhooks** [comment]
Although this pull request is already closed, I'm still watching it for updates.

There is nothing you have to do. No action is needed from your side.  Thanks.

**Pull request closed without merging**